### PR TITLE
order updates by dependency group and name

### PR DESCRIPTION
**Pull request by kamilszymanski**

### CODE REVIEW

**Comment by szpak** [comment]
Does it solve #30?

File: src/main/groovy/com/ofg/uptodate/finder/NewVersionFinderInAllRepositories.groovy [hunk 1]
```diff
@@ -43,7 +45,8 @@ class NewVersionFinderInAllRepositories {
         if (dependenciesWithNewVersions.isEmpty()) {
             loggerProxy.info(log, NO_NEW_VERSIONS_MESSAGE)
         } else {
-            loggerProxy.warn(log, NEW_VERSIONS_MESSAGE_HEAD + dependenciesWithNewVersions.join('\n'))
+            List<Dependency> sortedUpdates = dependenciesWithNewVersions.sort(new DependencyGroupAndNameComparator())
+            loggerProxy.warn(log, "$NEW_VERSIONS_MESSAGE_HEAD\n${sortedUpdates.join('\n')}")
```
Comment (szpak) [inline src/main/groovy/com/ofg/uptodate/finder/NewVersionFinderInAllRepositories.groovy]: warn? maybe [lifecycle](https://gradle.org/docs/current/userguide/logging.html) would be enough?

**Comment by szpak** [comment]
Nice enhancement!
+1

**Comment by kamilszymanski** [comment]
@szpak looks like it solves #30

File: src/main/groovy/com/ofg/uptodate/finder/NewVersionFinderInAllRepositories.groovy (hunk 1 above)
Comment (kamilszymanski) [inline src/main/groovy/com/ofg/uptodate/finder/NewVersionFinderInAllRepositories.groovy]: will make another PR for that
